src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1373,26 +1396,26 @@ private CorInfoTypeWithMod getArgType(IntPtr _this, CORINFO_SIG_INFO* sig, CORIN
             }
         }
 
-        private CorInfoType getHFAType(IntPtr _this, CORINFO_CLASS_STRUCT_* hClass)
+        private CorInfoType getHFAType(CORINFO_CLASS_STRUCT_* hClass)
         { throw new NotImplementedException("getHFAType"); }
-        private HRESULT GetErrorHRESULT(IntPtr _this, _EXCEPTION_POINTERS* pExceptionPointers)
+        private HRESULT GetErrorHRESULT(_EXCEPTION_POINTERS* pExceptionPointers)
         { throw new NotImplementedException("GetErrorHRESULT"); }
-        private uint GetErrorMessage(IntPtr _this, short* buffer, uint bufferLength)
+        private uint GetErrorMessage(short* buffer, uint bufferLength)
         { throw new NotImplementedException("GetErrorMessage"); }
 
-        private int FilterException(IntPtr _this, _EXCEPTION_POINTERS* pExceptionPointers)
+        private int FilterException(_EXCEPTION_POINTERS* pExceptionPointers)
         {
             return 0; // EXCEPTION_CONTINUE_SEARCH
```

Comment:
It would be nice to comment, assert and throw to make it clear that this method is not expected to be called because of it is implemented on the unmanaged side